front/app/typings.ts

```diff
@@ -99,6 +99,15 @@ export interface IOption {
   disabled?: boolean;
 }
 
+export function isOption(
+  maybeOption: {
+    value: string;
+    label: string;
+  } | null
+): maybeOption is IOption {
+  return maybeOption !== null;
+}
+
```

Comment:
maybeOption could be changed/extended in the future, just made it fit the use case I had.